kolibri/plugins/management/assets/src/views/class-edit-page/user-remove-modal.vue

```diff
@@ -72,13 +70,35 @@
         required: true,
       },
     },
+    computed: {
+      formattedDeleteConfirmation() {
+        return this.$tr(
+          'deleteConfirmation',
+          {
+            username: this.bold(this.username),
+            classname: this.bold(this.classname),
+          }
+        );
+      },
+      formattedAccessReassuranceConfirmation() {
+        return this.$tr(
+          'accessReassurance',
+          {
+            sectionTabName: this.bold(this.$tr('usersTab')),
+          }
+        );
+      },
+    },
     methods: {
       userRemove() {
         this.removeClassUser(this.classid, this.userid);
       },
       close() {
         this.displayModal(false);
       },
+      bold(stringToBold) {
+        return `<strong v-html> ${stringToBold} </strong>`;
```

Comment:
@indirectlylit not sure how I feel about this, but I did it to keep the translations free of markup. I figure it'll be cleaner for translators?